### DISCUSSION TITLE
Suggestion: make free rotation optional / additional "north up" mode

### DISCUSSION THREAD

**Discussion by wehkah** [osmandapp/OsmAnd#17393] [Ideas]
It is quite convenient to turn the map using two-finger-actions. But it you just want to enlarge or shrink the map using pinch or squeeze actions, and you accidentally turn your fingers during this gesture, the map will be rotated as well. Therefore, I suggest an additional "north up" mode without free rotation activated, so that north stays up, no matter how you move your fingers on the screen.

**Comment by mnalis** [comment]
which version are you using? There have been significant changes in 4.5.1, see https://github.com/osmandapp/OsmAnd/issues/17086#issuecomment-1569285378 which should be fixing most issues the people are having with 4.4.x

You may also be interested in #16836 and PR #17197

**Comment by wehkah** [comment]
Thanks for replying! I am using the latest version available at GooglePlayStore: 4.4.7. Your answer makes me looking forward to the next update! :)

Is there any kind of blog or diary, on which changes are listed in a way that "normal people" (=opposite of programmers) can understand?

**Reply by mikehgentry** [reply]
Best I've found is this, but it doesn't tell you what's upcoming, only what's just been done (except when they do a 'the year ahead' post, which unsurprisingly doesn't always turn out that accurate).

https://osmand.net/blog

You can keep an eye on the milestones to get a rough idea what they're doing. current+1-android and current+1-backend are the main ones to watch, but note that not *every* issue that gets solved will have been assigned to those milestones first.

https://github.com/osmandapp/OsmAnd/milestones

**Reply by mnalis** [reply]
To see what is released, apart from resources given by @mikehgentry, there is https://osmand.net/docs/versions/free-versions/ which gives in very terse notes the most popular changes. But it won't be shown until actual release.

Of course, the easiest way is to wait until new version pops up on GooglePlayStore and then just try that if that fixes the issue for you. That too requires patience, though :smile: 

If you're feeling a little more adventurous, you can download latest [Nighly build](https://download.osmand.net/latest-night-build/OsmAnd-default.apk)  (it installs alongside your regular GooglePlayStore version, so it should be OK to test). Then you can check if it fixes that issue adequately for you (and give your feedback if it does/doesn't -- it helps programmers to make better version of the app).

(Of course there is nothing preventing you to continue using Nightly if you like it; but it is usually not as polished as version released on app stores, and may contain other bugs, so people often uninstall it after testing what they wanted)
